driver/main.F90

```diff
@@ -152,9 +152,6 @@ program rdycore_f90
       PetscCallA(RDyGetLocalCellZCentroids(rdy_, n, values, ierr))
       PetscCallA(RDyGetLocalCellNaturalIDs(rdy_, n, nat_id, ierr))
 
-      PetscCallA(RDyGetLocalCellManningsNs(rdy_, n, values, ierr))
-      PetscCallA(RDySetManningsNForLocalCells(rdy_, n, values, ierr))
```

Comment:
Will you add the capabilities to get/set Mannings N from a Fortran code in a future PR?

Reply:
This didn't actually do anything, as far as I can tell--it just copied the mannings values out and back in again (right?).

Reply:
Yes, they only extracted or set values for ManningN. These functions could be needed for coupling with E3SM.

Reply:
These functions can be added back in a future PR.